tests/scripts/all.sh

```diff
@@ -88,6 +88,11 @@ elif [ -d library -a -d include -a -d tests ]; then :; else
     exit 1
 fi
 
+if ! [ -f crypto/Makefile ]; then
```

Comment:
Non-blocker: I'd like an option `--no-submodule` to say “I don't care about the submodule, just test mbedtls.”